File: src/main/java/com/angelhack/crowdtrust/entities/Organization.java
package com.angelhack.crowdtrust.entities;

import java.io.Serializable;
import java.util.List;

/**
 * Created by David Useche on 4/08/2018.
 */

public class Organization implements Serializable{

    String name;
    int id;
    List<Proyect> proyects;

}

File: src/main/java/com/angelhack/crowdtrust/entities/Organization.kt
package com.angelhack.crowdtrust.entities

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Created by David Useche on 4/08/2018.
 */


class Organization {

    @JsonProperty(value = "name")
    var name: String = ""

    @JsonProperty(value = "id")
    var id: Int = 0
    //var proyects: List<Project>
}
